Assets/Plugins/Source/Editor/EditorWindows/CheckDeploymentWindow.cs

```diff
@@ -101,7 +101,7 @@ protected override void RenderWindow()
 
         private void LoadTextFile(string filepath)
         {
-            content = FileUtility.ReadAllText(filepath);
+            content = FileSystemUtility.ReadAllText(filepath);
```

Comment:
Is it feasible to make this use the async version somehow?

Reply:
Possibly - there are many areas where we are using synchronous code where asynchronous code would arguably make more sense - however doing that often requires a lot more refactoring to do it properly. This is why we maintain synchronous calls. I'm not as familiar with the `CheckDeploymentWindow` code, so I am not sure what would be required. For the scope of this PR however, the intent is only to change the name of the `FileUtility` to `FileSystemUtility` to reflect the expanded functionality of the class.

Reply:
Valid. Room for improvement in another space, perhaps.